lib/chat_widget.dart

```diff
@@ -19,6 +22,35 @@ class SingleChat extends StatefulWidget {
 
 class _SingleChatState extends State<SingleChat> {
   bool actionsExpanded = false;
+  bool imageSelected = false;
+  late File image;
```

Comment:
this would be better as a File? vs a late.  Otherwise if this is ever used without being initialized it could still through an error (sort of an NPE)